### PR TITLE
Alternative installation server doc

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: docs/adminguide/hosting/index.rst [hunk 1]
```diff
@@ -241,6 +244,88 @@ You can test ``mailhog`` with:
 
 You can then check if the email is present in the web interface at http://127.0.0.1:8025
 
+On Bare Metal
+=============
+
+If you do want to install the parsec server through docker, you can install it bare metal.
```
Comment (mmmarcos) [inline docs/adminguide/hosting/index.rst:250]:
```suggestion
If you don't want to install Parsec server via Docker, you can install it on bare metal.
```

**Review by mmmarcos — COMMENTED** [review]
Thanks for these docs 🙏

File: docs/adminguide/hosting/index.rst [hunk 2]
```diff
@@ -241,6 +244,88 @@ You can test ``mailhog`` with:
...
+    pip install 'parsec-cloud[backend]==2.16.0-a.0+dev'
+
+4. Prepare the database by applying the migrations:
+
+  .. code-block:: bash
+
+    source venv/bin/activate
+    set -a
+    source parsec-db.env
+    python -m parsec.cli backend migrate
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply docs/adminguide/hosting/index.rst:297]: Yes

File: docs/adminguide/hosting/index.rst [hunk 3]
```diff
@@ -241,6 +244,88 @@ You can test ``mailhog`` with:
...
+
+If you do want to install the parsec server through docker, you can install it bare metal.
+
+Requirements
+------------
+
+- Python v3.9 with ``pip`` and ``venv`` modules
+- A S3 like object storage endpoint.
+- A postgres-14 database endpoint.
+- A TLS certificate & key for the server.
```
[the remark this answers is not in the document]
Comment (mmmarcos) [reply docs/adminguide/hosting/index.rst:258]: @Landeers what do you think? We can use the recommendations from the kickoff meeting.

[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply docs/adminguide/hosting/index.rst:258]: I understand why you would like to have the hardware requirement listed but consider:

1. The hardware requirement like depends on the usage of the server.
2. We don't have on our side a SIEM that would trace the usage of the server per user connected or something like that (I think of a Grafana graph). 

   With a SIEM that graph those data, we could obtain something like:

   - In term of CPU, we need ${cpu\\_per\\_user} * {users}$
   - In term of memory, we need ${ram\\_per\\_user} * {users}$

   Those value could also be obtain if we do a benchmark of the server :thinking:, but with the SIEM we would have a more realistic server load.

File: docs/adminguide/hosting/index.rst [hunk 4]
```diff
@@ -241,6 +244,88 @@ You can test ``mailhog`` with:
...
+    #!/bin/bash
+
+    # Load the virtualenv.
+    source venv/bin/activate
+
+    # Load the env file into the environment table.
+    set -a
+    for env_file in parsec-admin-token.env parsec-db.env parsec-smtp.env parsec-s3.env parsec.env; do
+      source $env_file
+    done
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply docs/adminguide/hosting/index.rst:315]: done

File: docs/adminguide/hosting/index.rst [hunk 5]
```diff
@@ -241,6 +244,88 @@ You can test ``mailhog`` with:
...
+
+    source venv/bin/activate
+    set -a
+    source parsec-db.env
+    python -m parsec.cli backend migrate
+
+Start the server
+----------------
+
+1. Create a wrapper script ``parsec-server-wrapper``
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply docs/adminguide/hosting/index.rst]: done

File: docs/adminguide/hosting/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmmarcos) [reply docs/adminguide/hosting/index.rst:258]: We do have estimates for cpu, ram and metadata/data storage based on our SaaS server. But you're right that it depends on type of usage. 

Maybe we can add that information later, after we have defined some standard use cases and give recommendations for those use cases.

**Review by mmmarcos — APPROVED** [review]

**Pull request merged**